sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -51,6 +54,10 @@ def evaluate(
         Used to get a score function that takes y_pred and y_test arguments
         and accept y_train as keyword argument.
         If None, then uses scoring = MeanAbsolutePercentageError(symmetric=True).
+    error_score : "raise" or numeric, default=np.nan
+        Value to assign to the score if an error occurs in estimator fitting. If set
```

Comment:
technical term is exception, not error

Reply:
(might be especially confusing, since some of the metrics are error metrics)

Reply:
the name `error_score` comes from `sklearn`. I can change it in the description however

Reply:
makes sense, keeping the arg consistent but clarifying the description